Code/Tools/AssetProcessor/native/AssetManager/SourceFileRelocator.cpp

```diff
@@ -413,11 +426,14 @@ Please note that only those seed files will get updated that are active for your
                 return AZ::Failure(AZStd::string("Cannot operate on directories.  Please specify a file or use a wildcard to select all files within a directory.\n"));
             }
 
-            GetFilesFromSourceControl(sources, scanFolderInfoOut, absoluteSourcePath.c_str(), excludeMetaDataFiles);
+            fileExists = GetFilesFromSourceControl(sources, scanFolderInfoOut, absoluteSourcePath.c_str(), excludeMetaDataFiles);
 
             if (sources.empty())
             {
-                return AZ::Failure(AZStd::string("File not found.\n"));
+                if (fileExists)
+                    return AZ::Failure(AZStd::string("File is not found in the asset database and will not be processed.\n"));
```

Comment:
is there any way to include the file pattern or any input that might have caused this to fail?